runtime/lua/vim/lsp/handlers.lua

```diff
@@ -441,7 +447,13 @@ M['textDocument/implementation'] = location_handler
 ---@param config table Configuration table.
 ---     - border:     (default=nil)
 ---         - Add borders to the floating window
----         - See |nvim_open_win()|
+---     - anchor_bias ("auto"|"above"|"below") defaults to "auto"
```

Comment:
Perhaps this should link to `|vim.lsp.util.make_floating_popup_options()|`. Duplicating non-trivial docs in many places, is hard to maintain.